explorer/templates.go

```diff
@@ -151,6 +151,39 @@ func float64Formatting(v float64, numPlaces int, useCommas bool, boldNumPlaces .
 	return []string{integer, dec[:places], dec[places:], trailingZeros}
 }
 
+func amountAsDecimalPartsTrimmed(v int64, numPlaces int64, useCommas bool) []string {
+	amt := strconv.FormatInt(v, 10)
+	if len(amt) <= 8 {
+		dec := strings.TrimRight(amt, "0")
+		trailingZeros := ""
+		leadingZeros := strings.Repeat("0", 8-len(amt))
+		if (8 - len(amt)) > 2 {
+			leadingZeros = "00"
+		}
+		return []string{"0", leadingZeros + dec, trailingZeros}
+	}
+	integer := amt[:len(amt)-8]
+	if useCommas {
+		integerAsInt64, err := strconv.ParseInt(integer, 10, 64)
+		if err != nil {
+			log.Errorf("amountAsDecimalParts comma formatting failed. Input: %v Error: %v", v, err.Error())
+			integer = "ERROR"
+			dec := "VALUE"
+			zeros := ""
+			return []string{integer, dec, zeros}
+		}
+		integer = humanize.Comma(integerAsInt64)
+	}
+	dec := strings.TrimRight(amt[len(amt)-8:], "0")
```

Comment:
This function is still broken is several ways.  Fix the test and reference my previous comment on how to simplify this function and make it less error prone: https://github.com/decred/dcrdata/pull/721#discussion_r222837818